src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme

```diff
@@ -37,7 +37,7 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "UITesting"
```

Comment:
Why UITesting in ENA? I thought we need it only for ENAUITests?

Reply:
Checking this locally, looks like we don't need it.